pkg/ingester/wal_test.go

```diff
@@ -2,19 +2,12 @@ package ingester
 
 import (
 	"io/ioutil"
-	"os"
 	"testing"
 	"time"
 
-	"github.com/cortexproject/cortex/pkg/util"
-	"github.com/go-kit/kit/log"
 	"github.com/stretchr/testify/require"
 )
 
-func init() {
```

Comment:
Just curious why you are removing this? Also I've noticed this a useful pattern in a lot of our tests and was curious if you had any thought's on initializing a logger during tests.

Reply:
I've re-introduced it so that we can merge this PR while discussing about the stdout logger in tests.